src/ec/packages/ec-component-site-header/site-header.html.twig

```diff
@@ -143,9 +138,20 @@
                 name: 'language'
               }
             } only %}
-            <span class="ecl-site-header__language-code">{{ _language.code }}</span>
+            <span class="ecl-site-header__language-code">{{ _language_selector.code }}</span>
           </span>
         </a>
+        {% include '../ec-component-language-list/language-list-overlay.html.twig' with { 
+          items: _language_selector.overlay.items,
+          close_label: _language_selector.overlay.close_label,
+          title: _language_selector.overlay.title,
```

Comment:
Why in ECL we get `/example#language_bg` as the href and here we have it empty..?